packages/shared/lib/services/notification/webhook.service.ts

```diff
@@ -4,12 +4,10 @@ import dayjs from 'dayjs';
 import utc from 'dayjs/plugin/utc.js';
```

Comment:
what still live in this file? Is everything moved over to the webhooks package?

Reply:
webhook forwarding logic which I'll bring over in another PR.